src/main/resources/schema/input.graphqls

```diff
@@ -1,5 +1,8 @@
-input WishInput {
+
+input WishInput{
+  author: ID!
```

Comment:
I think we can remove author id from wish input - we will know the author based on his/her authorization token